nsis/src/nsis/instructions/MessageBox.java

```diff
@@ -34,4 +36,37 @@ private void resolveStrings(Instruction instr, MemoryBlock stringsBlock)
         SourceType.ANALYSIS);
   }
 
+  /**
+   * Resolves the conditional branches
+   * 
+   * @param instr
+   * @param entriesBlock
+   * @param referenceManager
+   * @throws MemoryAccessException
+   */
+  private void resolveBranches(Instruction instr, MemoryBlock entriesBlock,
+      ReferenceManager referenceManager) throws MemoryAccessException {
+    instr.setFlowOverride(FlowOverride.BRANCH);
+
+    int arg4InstructionNumber = instr.getInt(NsisConstants.ARG4_OFFSET);
```

Comment:
Can we have a more descriptive variable name here? E.g. 'branchDestination1' and 'branchDestination2'